src/main/kotlin/no/nav/sosialhjelp/avtaler/digipost/DigipostClient.kt

```diff
@@ -44,11 +43,10 @@ class DigipostClient(props: Configuration.DigipostProperties, virksomhetProps: C
     private val virksomhetVersionId = virksomhetProps.versionId
     private val keyStoreConfig: KeyStoreConfig = configure(accessSecretVersion)
     private val clientConfiguration = ClientConfiguration.builder(keyStoreConfig)
-        .trustStore(if (profile == Configuration.Profile.PROD) Certificates.PRODUCTION else Certificates.TEST)
-        .serviceUri(if (profile == Configuration.Profile.PROD) ServiceUri.PRODUCTION else ServiceUri.DIFI_TEST)
-        .globalSender(Sender(props.navOrgnr))
-        .enableRequestAndResponseLogging()
```

Comment:
ServiceEnvironment ser ut til å holde på ServiceUri og Certificate.
globalSender ser ut til å ha blitt erstattet av defaultSender.
enableRequestAndResponseLogging kan jeg ikke se noe sted lengre

Reply:
Jeg vet ikke om vi trenger `enableRequestAndResponseLogging`, tror jeg la den på for å debugge så vi kan se om vi klarer oss uten.

Reply:
Etter at de gikk vekk fra Jersey, kan dette configureres litt ulikt:
https://github.com/digipost/signature-api-client-java/commit/61a84e7bb01139048db8438cf4a1e92f32d2807e